### ISSUE TITLE
when chef version not set terraform fails

### ISSUE THREAD

**Issue by ramonskie** [hashicorp/terraform#2568]
when using the chef provisioner
and not provide the version as stated in
https://www.terraform.io/docs/provisioners/chef.html#version_string_
it should use the latest

but in stead it just fails because it still provides the -v option

```
* Command "sudo bash ./install.sh -v " exited with non-zero exit status: 1
```
see https://github.com/hashicorp/terraform/blob/master/builtin/provisioners/chef/linux_provisioner.go#L36

**Comment by svanharmelen** [comment]
Thanks for the report @ramonskie! Just fixed this one in PR #2570.

**Comment by ramonskie** [comment]
:+1:
